validation-test/stdlib/Arrays.swift.gyb

```diff
@@ -158,8 +158,7 @@ ArrayTestSuite.test("${Kind}/append") {
 }
 % end
 
-ArrayTestSuite.test("ArraySlice/append") 
-  .xfail(.custom({true}, reason: "<rdar://problem/33997561>")).code {
+ArrayTestSuite.test("ArraySlice/append") {
```

Comment:
This could be an issue when building with a debug stdlib. There is a known issue where we are not properly scoping arguments causing left-right evaluation order to be broken and thus breaking uniqueness checks. The optimizer is good enough that it eliminates the problem with an optimized stdlib and we get back to the normal no copy behavior. I am actively fixing this. So if it was fixed, but now it is not 100% clear, I would check that.